src/Suave/Suave.fsproj

```diff
@@ -196,4 +196,89 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Choose>
```

Comment:
The underlying reason is that it will cause assembly binding conflicts, because of the frequency that Newtonsoft.Json is being used. Also, it's very backwards-incompatible for consumers who don't have the latest newtonsoft.